test-cloud/pom.xml

```diff
@@ -467,18 +467,36 @@ $ keytool -import -trustcacerts -alias name -keypass pass -storepass pass -keyst
     </profile>
     <profile>
       <id>openshift</id>
+      <properties>
+        <template.excluded.group>org.kie.cloud.integrationtests.category.TemplateNotSupported</template.excluded.group>
+      </properties>
       <dependencies>
         <dependency>
           <groupId>org.kie.cloud</groupId>
           <artifactId>framework-openshift-templates</artifactId>
           <scope>test</scope>
         </dependency>
       </dependencies>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-failsafe-plugin</artifactId>
+              <configuration>
+                <excludedGroups>${template.excluded.group},${failsafe.excluded.groups}</excludedGroups>
+              </configuration>
+            </plugin>
+          </plugins>
+        </pluginManagement>
+      </build>
     </profile>
     <profile>
       <id>openshift-operator</id>
       <properties>
         <operator.excluded.group>org.kie.cloud.integrationtests.category.OperatorNotSupported</operator.excluded.group>
+        
+        <kie.operator.upgrade.from.image.tag>7.6.0</kie.operator.upgrade.from.image.tag>
```

Comment:
Can you please move this version declaration to https://github.com/kiegroup/kie-cloud-tests/blob/master/pom.xml#L28 to be among other versions?
That way we can change versions just in one place.

Reply:
Ok, will do.